rebac-admin-backend/v1/identities.go

```diff
@@ -0,0 +1,176 @@
+	identity := new(resources.Identity)
+	defer req.Body.Close()
+
+	err := json.NewDecoder(req.Body).Decode(identity)
+	if err != nil {
+		writeErrorResponse(w, NewValidationError("Request doesn't match the expected schema"))
+		return
+	}
+
+	identity, err = h.Identities.CreateIdentity(identity)
+	if err != nil {
+		response := h.IdentitiesErrorMapper.MapError(err)
+		writeResponse(w, response.Status, response)
+		return
+	}
+
+	writeResponse(w, 201, identity)
+}
+
+// DeleteIdentitiesItem deletes the specified identity.
+// (DELETE /identities/{id})
+func (h handler) DeleteIdentitiesItem(w http.ResponseWriter, req *http.Request, id string) {
+	_, err := h.Identities.DeleteIdentity(id)
+	if err != nil {
+		response := h.IdentitiesErrorMapper.MapError(err)
+		writeResponse(w, response.Status, response)
+		return
+	}
+
+	w.WriteHeader(200)
+}
+
+// GetIdentitiesItem returns the identity identified by the provided ID.
+// (GET /identities/{id})
+func (h handler) GetIdentitiesItem(w http.ResponseWriter, req *http.Request, id string) {
+	identity, err := h.Identities.GetIdentity(id)
+	if err != nil {
+		response := h.IdentitiesErrorMapper.MapError(err)
+		writeResponse(w, response.Status, response)
+		return
+	}
+
+	writeResponse(w, 200, identity)
+}
+
+// PutIdentitiesItem updates the identity identified by the provided ID.
+// (PUT /identities/{id})
+func (h handler) PutIdentitiesItem(w http.ResponseWriter, req *http.Request, id string) {
+	identity := new(resources.Identity)
+	defer req.Body.Close()
+
+	err := json.NewDecoder(req.Body).Decode(identity)
+	if err != nil {
+		writeErrorResponse(w, err)
+		return
+	}
+
+	if id != *identity.Id {
+		writeErrorResponse(w, NewValidationError("Identity ID from path does not match the Identity object"))
+		return
+	}
+
+	identity, err = h.Identities.UpdateIdentity(identity)
+	if err != nil {
+		response := h.IdentitiesErrorMapper.MapError(err)
+		writeResponse(w, response.Status, response)
+		return
+	}
+
+	writeResponse(w, 200, identity)
+}
+
+// GetIdentitiesItemEntitlements returns the list of entitlements for an identity identified by the provided ID.
+// (GET /identities/{id}/entitlements)
+func (h handler) GetIdentitiesItemEntitlements(w http.ResponseWriter, req *http.Request, id string, params resources.GetIdentitiesItemEntitlementsParams) {
+	entitlements, err := h.Identities.GetIdentityEntitlements(id, &params)
+	if err != nil {
+		response := h.IdentitiesErrorMapper.MapError(err)
+		writeResponse(w, response.Status, response)
+		return
+	}
+
+	response := resources.GetIdentityEntitlementsResponse{
+		Data:   entitlements.Data,
+		Status: 200,
+	}
+
+	writeResponse(w, 200, response)
+}
+
+// PatchIdentitiesItemEntitlements Adds or removes entitlements to/from an identity.
+// (PATCH /identities/{id}/entitlements)
+func (h handler) PatchIdentitiesItemEntitlements(w http.ResponseWriter, req *http.Request, id string) {
+	w.WriteHeader(http.StatusNotImplemented)
```

Comment:
Please create a card for the upcoming PR, and add a `// TODO([card-id])` comment here.